mcux/README

```diff
@@ -4,13 +4,15 @@ MCUXpresso SDK
 Origin:
    NXP MCUXpresso SDK
    https://github.com/NXPmicro/mcux-sdk
+   SHA: 076dc8823ac7c8c5499688418aa7691160a10ec6
 
    NXP Kinetis Connectivity Software
    https://www.nxp.com/webapp/Download?colCode=KW40Z-CONNECTIVITY-SOFTWARE&appType=license&Parent_nodeId=1432854896956716810497&Parent_pageType=product
    https://www.nxp.com/webapp/Download?colCode=MKW2XD-MRC20A-CONNECTIVITY-SW&appType=license&Parent_nodeId=1425322332576706617013&Parent_pageType=productI
 
    NXP MCUXpresso SDK USB
    https://github.com/NXPmicro/mcux-sdk-middleware-usb
+   SHA: c8317a37546b18bef2072d6075e061d668b3f1c7
```

Comment:
I don't think [mcux-sdk-middleware-usb](https://github.com/nxp-mcuxpresso/mcux-sdk-middleware-usb) `main` has been updated with commits from the release associated with MCUX SDK v2.15.000.

Do you need to use [this](https://github.com/nxp-mcuxpresso/mcux-sdk-middleware-usb/releases/tag/MCUX_2.15.000) tagged release instead? (Though then you'll miss a recent fix in `main` :/)

Reply:
The NXP SDK team is starting to shift to a release to main approach. This sha is supposed to be 2.15.000+updates to support the RW61x SOCs

Reply:
@brycewilkins, a separate HAL PR was merged that included the middleware-usb

https://github.com/zephyrproject-rtos/hal_nxp/pull/316